src/eq_schema/Block.test.js

```diff
@@ -34,4 +34,40 @@ describe("Block", () => {
       ]
     });
   });
+
+  describe("conversion of page types", () => {
+    it("should convert QuestonPage to Questionnaire", () => {
+      const authorJson = {
```

Comment:
If you declare this out of the scope of this function you won't need to redeclare in each `it`. You can then reassign value later on.